plugins/woocommerce-blocks/assets/js/blocks/checkout/inner-blocks/checkout-actions-block/constants.tsx

```diff
@@ -3,4 +3,7 @@
  */
 import { __ } from '@wordpress/i18n';
 
-export const defaultPlaceOrderButtonLabel = __( 'Place Order', 'woocommerce' );
+export const defaultPlaceOrderButtonLabel = __(
+	'Place Order · <price/>',
+	'woocommerce'
+);
```

Comment:
```suggestion
export const defaultPlaceOrderButtonLabel = sprintf(
	// translators: %s: is the price.
	__( 'Place Order · %s', 'woocommerce' ),
	'<price/>'
);
```

This is to ensure a translator doesn't mistakenly change how it's worded there and follows the standard of placeholders.

Reply:
This makes sense. Thanks for this suggestion.